s3upload.js

```diff
@@ -59,7 +59,8 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
-    xhr.open('GET', this.signingUrl + '?objectName=' + file.name, true);
+    var fileName = file.name.replace(/[^\w]/g, "");
```

Comment:
Would it work to use something like an underscore instead of removing the space?  Thinking the space in a file name might want to be mildly preserved.